app/backend/src/tests/test_db.py

```diff
@@ -162,7 +162,7 @@ def test_migrations(testconfig):
 
     from_scratch = pg_dump()
 
-    def massage(s):
+    def message(s):
```

Comment:
This was not a typo, but I realised I could have used a better verb.

Reply:
Oof 🤦‍♂️ Sorry about that. Happy to fix unless you have already, just lmk

Reply:
Nothing to worry about :)